crates/ruff_python_parser/resources/inline/ok/args_unparenthesized_generator.py

```diff
@@ -1 +1,2 @@
 sum(x for x in range(10))
+sum((x for x in range(10),)
```

Comment:
I think this is the cause of the CI failure:
```suggestion
sum((x for x in range(10)),)
```

Reply:
Oops, looks like a `git add` miss on my end. Fixed.